src/components/MyKiva/BadgesSectionV2.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+	<KvCarousel
+		ref="carousel"
+		class="tw-w-full"
```

Comment:
Do we need a `key` attribute here if number of slides changes?

Reply:
keys are defined in template bellow